tests/nexus_api_test.go

```diff
@@ -440,17 +442,16 @@ func (s *ClientFunctionalSuite) TestNexusStartOperation_Claims() {
 		capture := s.testCluster.host.captureMetricsHandler.StartCapture()
 		defer s.testCluster.host.captureMetricsHandler.StopCapture(capture)
 
-		result, err := nexus.StartOperation(ctx, client, op, "input", nexus.StartOperationOptions{
-			Header: tc.header,
-		})
-
-		snap := capture.Snapshot()
-		tc.assertion(t, result, err, snap)
+		s.EventuallyWithT(func(c *assert.CollectT) {
```

Comment:
From the examples in the docs using `EventuallyWithT` I saw that they were using `assert` and not `require` in the block. `tc.assertion` functions are using `require`. Can you verify that `require` is compatible with `EventuallyWithT`?